src/index.html

```diff
@@ -17,16 +17,42 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="forma forma__big"
```

Comment:
```suggestion
      class="form form__big"
```
Use only English words for names